public/index.html

```diff
@@ -1,50 +1,57 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
     <link rel="stylesheet" href="./client.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-
+    <h1>Account</h1>
     <div data-hide="false" class="generic-loader-wrap"></div>
 
     <section data-hide="true" class="input-forms">
-      <h3>Sign In</h3><br>
-      <input name="login-username-field" type="text" placeholder="Username"><br>
-      <input name="login-password-field" type="password" placeholder="Password"><br>
+      <h2>Sign In</h2><br>
+      <label>USERNAME</label><br>
+      <input name="login-username-field" type="text"><br>
+      <label>PASSWORD</label><br>
+      <input name="login-password-field" type="password"><br>
```

Comment:
two things:

- avoid using uppercase only text because screen readers will read it out letter by letter, you can use css for that ([`text-transform: uppercase`](https://developer.mozilla.org/en/docs/Web/CSS/text-transform))
- add the `for` attribute to labels and `id` attributes to the respective inputs, with same values, e.g. 

```html
<label for="login-username-field">Username</label><br>
<input name="username" type="text" id="login-username-field"><br>
```